module/nilai.go

```diff
@@ -0,0 +1,54 @@
+package module
+
+import (
+	"context"
+
+	nilai "github.com/tokopedia/code-review-workshop/core/nilai"
+	"github.com/tokopedia/tokopoints/errors"
+	"github.com/tokopedia/tokopoints/tracer"
+)
+
+func GetNilaiSiswa(ctx context.Context, nim int64, sekolah, kelas string, mapel_id int64) (resp GetNilaiSiswaResponse, err error) {
+	span, ctx := tracer.StartSpanFromContext(ctx)
+	defer span.Finish()
+
+	if nim != 0 && mapel_id != 0 {
```

Comment:
```suggestion
	if nim > 0 && mapel_id > 0 {
```
antisipasi negative value